omnigibson/object_states/particle_modifier.py

```diff
@@ -550,7 +570,7 @@ def _modify_particles(self, system):
             if system.n_particles > 0:
                 # Iterate over all particles and remove any that are within the relaxed AABB of the remover volume
                 particle_names = list(system.particles.keys())
-                particle_positions = np.array([particle.get_position() for particle in system.particles.values()])
+                particle_positions = np.array([system.get_particle_position_orientation(name=name) for name in system.particles.keys()])
```

Comment:
should be `system.get_particle_position_orientations(name=name)[0]`?

Reply:
done!